kmk/kmk_keyboard.py

```diff
@@ -200,49 +213,75 @@ def go(self, hid_type=HIDModes.USB, **kwargs):
 
         self._state = InternalState(self)
 
-        if hid_type == HIDModes.NOOP:
-            self.hid_helper = AbstractHID
-        elif hid_type == HIDModes.USB:
-            try:
-                from kmk.hid import USBHID
-
-                self.hid_helper = USBHID
-            except ImportError:
-                self.hid_helper = AbstractHID
-                print('USB HID is unsupported ')
-        elif hid_type == HIDModes.BLE:
-            try:
-                from kmk.ble import BLEHID
-
-                self.hid_helper = BLEHID
-            except ImportError:
-                self.hid_helper = AbstractHID
-                print('Bluetooth is unsupported ')
-
-        self._hid_helper_inst = self.hid_helper(**kwargs)
-
-        # Split keyboard Init
         if self.split_type is not None:
-            try:
-                # Working around https://github.com/adafruit/circuitpython/issues/1769
-                self._hid_helper_inst.create_report([]).send()
-                self.is_target = True
-
-                # Sleep 2s so target portion doesn't "appear" to boot quicker than
-                # dependent portions (which will take ~2s to time out on the HID send)
-                sleep_ms(2000)
```

Comment:
This is no longer needed as boot times do not appear to be noticeably mismatched. Test more if you will, but I found it to feel better removed.